packages/elements/src/photon-client/index.tsx

```diff
@@ -104,6 +106,13 @@ customElement(
       reflect: false,
       notify: false,
       parse: false
+    },
+    externalUserId: {
+      attribute: 'userId',
+      value: undefined,
+      reflect: true,
```

Comment:
I would think [reflect](https://open-wc.org/guides/knowledge/attributes-and-properties/#attribute-and-property-reflection) would work, or maybe I'm misreading it?

Reply:
Yeah I think unrelated?